src/App.js

```diff
@@ -37,12 +37,20 @@ function App({ totalPosts = TOTAL_POSTS, pageLimit = PAGE_DEFAULT_LIMIT }) {
       });
   }
 
+  const getUser = (userId) => () => {
```

Comment:
What do you think about moving all the data fetching from here to a separate file?